data/locales.js

```diff
@@ -0,0 +1,29 @@
+export const locales = [
+  { code: "ar", language: "العربية" },
+  { code: "ca", language: "Català" },
+  { code: "cs", language: "Čeština" },
+  { code: "cy", language: "Cymraeg" },
+  { code: "de", language: "Deutsch" },
+  { code: "en", language: "English" },
+  { code: "es", language: "Español" },
+  { code: "eu", language: "Euskara" },
+  //{ code: 'fi', language: 'Suomi' },
+  { code: "fr", language: "Français" },
+  //{ code: 'he', language: 'עברית' },
+  { code: "it", language: "Italiano" },
+  { code: "ja", language: "日本語" },
+  { code: "ko", language: "한국어" },
+  { code: "nl-NL", language: "Nederlands" },
+  //{ code: 'no', language: 'Norsk' },
+  { code: "pl", language: "Polski" },
+  { code: "pt-BR", language: "Português" },
+  { code: "ru", language: "Русский" },
+  //{ code: 'sk', language: 'Slovenčina' },
+  { code: "sq", language: "Shqip" },
+  //{ code: 'sv', language: 'Svenska' },
+  { code: "th", language: "ภาษาไทย" },
+  { code: "tr", language: "Türkçe" },
+  //{ code: 'uk', language: 'Українська' },
+  { code: "zh", language: "中文" },
+  { code: "zh-TW", language: "繁體中文" },
+]
```

Comment:
good idea, we can prob import this data into [next.config.js](https://github.com/oakstudios/joinmastodon/blob/461b65b7ef57576b6d74ef5ee0e34521d7e81b09/next.config.js#L5-L32) for the locales list (that way we're not writing it twice)

Reply:
oo ty i didn't realize the config lived here too